src/main/kotlin/no/nav/faktureringskomponenten/service/avregning/AvregningBehandler.kt

```diff
@@ -17,7 +17,10 @@ private data class FakturaOgNyePerioder(val faktura: Faktura, val nyePerioder: L
 
 @Component
 class AvregningBehandler(private val avregningsfakturaGenerator: AvregningsfakturaGenerator) {
-    fun lagAvregningsfaktura(fakturaseriePerioder: List<FakturaseriePeriode>, bestilteFakturaer: List<Faktura>): Faktura? {
+    fun lagAvregningsfaktura(
+        fakturaseriePerioder: List<FakturaseriePeriode>,
+        bestilteFakturaer: List<Faktura>
+    ): List<Faktura>? {
         if (bestilteFakturaer.isEmpty()) return null
```

Comment:
Tom liste i stedet for `null` som returverdi for bedre semantikk og for å unngå unødvendige nullsjekker